runtime/oti/j9java8packages.h

```diff
@@ -0,0 +1,898 @@
+	J9_DECLARE_CONSTANT_UTF8_VALUE("javax.xml.ws.spi")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("javax.xml.ws.wsaddressing")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("javax.xml.ws")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("javax.xml.xpath")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("javax.xml")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.apache.harmony.kernel.vm")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.ietf.jgss")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CORBA.DynAnyPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CORBA.ORBPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CORBA.TypeCodePackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CORBA.portable")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CORBA")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CORBA_2_3.portable")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CORBA_2_3")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CosNaming.NamingContextExtPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CosNaming.NamingContextPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.CosNaming")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.Dynamic")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.DynamicAny.DynAnyFactoryPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.DynamicAny.DynAnyPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.DynamicAny")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.IOP.CodecFactoryPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.IOP.CodecPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.IOP")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.Messaging")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableInterceptor.ORBInitInfoPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableInterceptor")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableServer.CurrentPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableServer.POAManagerPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableServer.POAPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableServer.ServantLocatorPackage")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableServer.portable")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.PortableServer")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.SendingContext")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.omg.stub.java.rmi")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.bootstrap")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.css")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.events")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.html")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.ls")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.ranges")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.stylesheets")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.traversal")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.views")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom.xpath")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.w3c.dom")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.xml.sax.ext")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.xml.sax.helpers")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("org.xml.sax")
+	J9_DECLARE_CONSTANT_UTF8_VALUE("sun.reflect")
```

Comment:
How do we keep this list up to date if a new package is added to Java 8?  Is this list spec'd somewhere?

Reply:
AFAIK there is no spec list. This list includes all the packages in OpenJDK8 + J9 java8 packages.   
  
We would have to monitor the every Java8 release to make sure the list is up to date. 

Hopefully we won't have to do this for too long because there is talk that this functionality may be removed within the few releases.

Reply:
Java 8 is an LTS release and is likely to be supported for 3 years.  I don't relish having to validate this list for the next 3 years if we can come up with a better solution.

Javac now supports a `-release` flag which validates APIs are available in the targeted release to prevent accidentally compiling Java 9 APIs into something that should target Java 8.  This implies there is a data file somewhere with this information.  Can we reuse that file to find the set of Java 8 packages?

Reply:
Java9 Javac does not respect the --illegal-access=permit behaviour. What this means is that javac will not allow you to compile something that uses internal packages without explicit exports, opens and requires clauses in the module-info. So javac doesn't need to know what packages existed in java8.   
  
This makes sense as the --illegal-access=permit functionality was intended for migration purposes (java8 and older jars running on java9) and not for new code written with Java9.